templates/layouts/bootstrap.html.ep

```diff
@@ -36,10 +36,8 @@
           } );
       % end
 
-      <link rel="icon"
-            type="image/png" sizes="16x16"
-            href="<%= icon_url 'logo-16.png' %>">
-      <link rel="icon" href="<%= icon_url 'logo.svg'%>" sizes="any" type="image/svg+xml">
+      <link rel="icon" href="<%= favicon_url "-16.png"%>" sizes="16x16" type="image/png">
```

Comment:
A space between the last quote and `%>` would be nice. I'd also use single quotes here (especially because the HTML already uses double quotes here).

Reply:
done